catalyst/core/callbacks/checkpoint.py

```diff
@@ -120,20 +120,26 @@ def __init__(
         resume: str = None,
         resume_dir: str = None,
         metrics_filename: str = "_metrics.json",
-        load_on_stage_end: str = None,
+        load_on_stage_end: str = "best",
     ):
         """
         Args:
-            save_n_best (int): number of best checkpoint to keep
+            save_n_best (int): number of best checkpoint to keep,
+                if ``0`` then  store only last state of model and
+                ``load_on_stage_end`` should be one of
+                ``last`` or ``last_full``.
             resume (str): path to checkpoint to load
                 and initialize runner state
+            resume_dir (str): directory with checkpoints,
+                if specified in combination with ``resume``
+                than resume checkpoint will be loaded from ``resume_dir``
             metrics_filename (str): filename to save metrics
                 in checkpoint folder. Must ends on ``.json`` or ``.yml``
             load_on_stage_end (str): name of the model to load
                 at the end of the stage.
-                You can use ``best`` to load the best model according
-                to validation metrics, or ``last`` to use just the last one
-                (default behaviour).
+                You can use ``best``, ``best_full`` (default)
```

Comment:
looks like we need to update docs:
```
load_on_stage_end: str = "best",
```
and 
```
You can use ``best``, ``best_full`` (default)
```
which default is correct?

Reply:
Hmm, I think that more correct if we keep default value `load_on_stage_end` is `None`.
Why - it will be more appropriate to keep consistent with master branch.

Reply:
I see, let's refactor them later then